vulnerabilities/api.py

```diff
@@ -71,36 +73,64 @@ class MinimalVulnerabilitySerializer(serializers.HyperlinkedModelSerializer):
 
     class Meta:
         model = Vulnerability
-        fields = ["url", "vulnerability_id", "references", "summary"]
+        fields = ["url", "vulcoid", "summary", "references"]
+
+
+class AliasSerializer(serializers.HyperlinkedModelSerializer):
+    """
+    Used for nesting inside package focused APIs.
+    """
+
+    class Meta:
+        model = Alias
+        fields = ["alias"]
 
 
 class VulnerabilitySerializer(serializers.HyperlinkedModelSerializer):
 
-    resolved_packages = MinimalPackageSerializer(many=True, source="resolved_to", read_only=True)
-    unresolved_packages = MinimalPackageSerializer(
-        many=True, source="vulnerable_to", read_only=True
-    )
+    fixed_packages = MinimalPackageSerializer(many=True, source="resolved_to", read_only=True)
```

Comment:
resolved_to will need to be renamed separately... fixed_by could be a much better name. That's for another PR though

Reply:
Opened a follow-up issue here #715

Reply:
Opened a follow-up issue here #715